src/styles/blocks/reset.scss

```
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
```

```suggestion
  font-family: Arial, sans-serif;
```

move to html selector

---

what is it?

---

html {
  font-family: Arial, sans-serif;
}